pkg/agent/testdata/RawUbuntuContainerd/line62.sh

```diff
@@ -60,16 +60,7 @@ installDeps() {
 }
 
 updateAptWithMicrosoftPkg() {
-    if [[ $(isARM64) == 1 ]]; then
```

Comment:
where is corresponding change in for pkg/templates/templates_generated.go?

Reply:
I'm wondering that myself too
@anujmaheshwari1 @cameronmeissner Do you know if there is a separate command to run, than `make generate`

Reply:
Seems expected, discussed offline (and in [this PR](https://github.com/Azure/AgentBaker/pull/4434))